script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt

```diff
@@ -569,7 +569,7 @@ That would %{i}definitely%{/i} get me noticed."
 -- Page 120, Offset 39757.
 -- 『無論、正面は避け、迂回するべきだ。
 %{i}
-"You ought naturally avoid going straight,
+"You ought to naturally avoid going straight,
```

Comment:
Revert, speech style

Reply:
Yeah, this is like the third time. I should have known it was.